usaspending_api/common/management/commands/resave_instances.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+import logging
+import os
+from time import time
+
+from django.core.management.base import BaseCommand
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management import call_command
+from django.db import connection
+
+import django.db.models.base
+from django.apps import apps
+
+
+def models_with_custom_save():
+
+    result = [m for m in apps.get_models() if m.save != django.db.models.base.Model.save]
```

Comment:
Just for my knowledge, this is where Python is able to see that the `.save` was overridden?